app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render partial: 'shared/navbar' %>
+ <div class="flex flex-col gap-5 w-full md:w-[60%] mx-auto">
+  <% @recipes.each do |recipe| %>
+   <%= render partial: 'shared/recipe_card', locals: { recipe: recipe } %>
+  <% end %>
+  <%= link_to 'New Recipe', new_recipe_path %>
+</div>
```

Comment:
- You have done well so far, but according to the project requirement, your design should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true). This would give confidence in recruiters in your ability to match given business requirements. Please fix this 🙏 
![vaecae](https://github.com/AbdelazizSoliman/Recipe_app/assets/73607512/4d87273c-a649-4f33-9d1b-06f50c3fc5a9)